test/integration/config-gcs.yml

```diff
@@ -3,15 +3,30 @@ general:
   remote_storage: gcs
   upload_concurrency: 4
   download_concurrency: 4
+  skip_tables:
+    - " system.*"
+    - "INFORMATION_SCHEMA.*"
+    - "information_schema.*"
+    - "_temporary_and_external_tables.*"
   restore_schema_on_cluster: "{cluster}"
 clickhouse:
+  # wrong disk name mapping for https://github.com/Altinity/clickhouse-backup/issues/676
+  disk_mapping:
+    default-gp3: /var/lib/clickhouse
   host: clickhouse
-  port: 9000
-  restart_command: bash -c 'echo "FAKE RESTART"'
+  port: 9440
+  username: backup
+  password: meow=& 123?*%# МЯУ
+  secure: true
+  skip_verify: true
+  sync_replicated_tables: true
+  timeout: 5s
+  restart_command: "sql:SYSTEM RELOAD USERS; sql:SYSTEM RELOAD CONFIG; exec:ls -la /var/lib/clickhouse/access; sql:SYSTEM SHUTDOWN"
+  # restart_command: bash -c 'echo "FAKE RESTART"'
+  backup_mutations: true
 gcs:
   bucket: altinity-qa-test
   path: backup/{cluster}/{shard}
   object_disk_path: object_disks/{cluster}/{shard}
-  credentials_file: /etc/clickhouse-backup/credentials.json
```

Comment:
Please, rollback changes in this file
and use different `config-gcs-facegcs.yml`

`TestIntegrationGCS` used credentials from ./test/integration/credentials.json which decrypted with https://github.com/sobolevn/git-secret

Reply:
oups sorry for that, I was under impression that this test is always ignored. Reverted